operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -152,10 +152,9 @@ type StorageConfig struct {
 
 // GetRacks is a getter for the Rack slice in the spec
 // It ensures there is always at least one rack
-// FIXME move this onto the CassandraDatacenter for consistency?
-func (s *CassandraDatacenterSpec) GetRacks() []Rack {
-	if len(s.Racks) >= 1 {
-		return s.Racks
+func (dc *CassandraDatacenter) GetRacks() []Rack {
```

Comment:
I made this change in one of my PRs.  I think it's in master already, so you may want to merge in master to your PR.